### PR TITLE
Fix all instances of broken links to create-cagg

### LINKED ISSUE #1233: [Site Bug] Issue with the page: /timescaledb/latest/quick-start/node/#next-steps
_Use this template for reporting bugs in the docs._

# Describe the bug

The link to Continuous Aggregates at the bottom of the page incorrectly links to a [broken page](https://docs.timescale.com/timescaledb/latest/getting-started/create-cagg/).  It appears that it should instead link to [this page](https://docs.timescale.com/getting-started/latest/create-cagg/).

## What do the docs say now?
link to broken page

## What should the docs say?
link to working page

# Page affected
https://docs.timescale.com/timescaledb/latest/quick-start/node/#next-steps

# Version affected
latest

# Subject matter expert (SME)
n/a

# Screenshots
image

# Any further info
n/a

### PR DESCRIPTION
**Pull request by andy2mrqz**
# Description

I created issue timescale/docs#1233 because I encountered broken links to the getting started page for Creating a continuous aggregate.  I created this PR to resolve all broken links to the affected page.

# Links

Fixes timescale/docs#1233

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Are procedure and highlight tags used appropriately?
- [ ] Has the index been updated appropriately?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] Are all links provided in reference style, and resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

**Comment by andy2mrqz** [comment]
I checked that each of the links I have suggested fixes for were indeed broken on the current website. I have updated most of the links to either include the version or have the base changed to the `getting-started` root instead of `timescaledb`, according to the guidelines in the README.

Let me know if additional comments/links to the pages containing broken links would help you review this PR, and I would be happy to do so.

File: timescaledb/tutorials/analyze-cryptocurrency-data.md [hunk 1]
```diff
@@ -637,6 +637,6 @@ Ready for even more learning? Here's a few suggestions:
 [cryptocompare-apikey]: https://min-api.cryptocompare.com
 [tableau-tutorial]: /tutorials/visualize-with-tableau/
 [time-series-forecasting]: /tutorials/time-series-forecast/
-[continuous-aggregates]: /getting-started/create-cagg
```
Comment (andy2mrqz) [inline timescaledb/tutorials/analyze-cryptocurrency-data.md:640]: Page with currently broken link:
https://docs.timescale.com/timescaledb/latest/tutorials/analyze-cryptocurrency-data/#next-steps

File: timescaledb/quick-start/node.md [hunk 2]
```diff
@@ -519,7 +519,7 @@ Node application, be sure to check out these advanced TimescaleDB tutorials:
 [install-timescale]: /install/latest/
 [setup-psql]: /how-to-guides/connecting/psql/
 [time-series-forecasting]: /tutorials/time-series-forecast/
-[continuous-aggregates]: /getting-started/create-cagg/
```
Comment (andy2mrqz) [inline timescaledb/quick-start/node.md:522]: Page with currently broken link:
https://docs.timescale.com/timescaledb/latest/quick-start/node/#next-steps

File: timescaledb/overview/release-notes/index.md [hunk 3]
```diff
@@ -1439,7 +1439,7 @@ the materialized aggregate the next time that the automated process executes.
 
 For more information on this release, read our [blog on continuous aggregates](https://blog.timescale.com/blog/continuous-aggregates-faster-queries-with-automatically-maintained-materialized-views/),
 [our docs overview](/timescaledb/:currentVersion:/overview/core-concepts/continuous-aggregates/),
-and visit this [tutorial](/timescaledb/:currentVersion:/getting-started/create-cagg/).
```
Comment (andy2mrqz) [inline timescaledb/overview/release-notes/index.md:1442]: Page with currently broken link:
https://docs.timescale.com/timescaledb/latest/overview/release-notes/#v1-3-0-2019-05-06

File: timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md [hunk 4]
```diff
@@ -179,4 +179,3 @@ For more information about setting up multi-node, see the
 [multi-node]: /how-to-guides/multinode-timescaledb/
 [postgres-parallel-agg]: https://www.postgresql.org/docs/current/parallel-plans.html#PARALLEL-AGGREGATION
 [real-time-aggs]: /how-to-guides/continuous-aggregates/real-time-aggregates/
-[tutorial-caggs]: /getting-started/create-cagg
```
Comment (andy2mrqz) [inline timescaledb/how-to-guides/continuous-aggregates/about-continuous-aggregates.md:182]: This link was not referenced on this page.

File: cloud/services/index.md [hunk 5]
```diff
@@ -42,7 +42,7 @@ For additional documentation on how to:
 [hypertable-info]: /timescaledb/latest/how-to-guides/hypertables
 [time-bucket-info]: /timescaledb/latest/how-to-guides/query-data/advanced-analytic-queries#time-bucket
 [gap-filling-info]: /timescaledb/latest/how-to-guides/query-data/advanced-analytic-queries#gap-filling
-[aggregates-info]: /timescaledb/latest/getting-started/create-cagg
```
Comment (andy2mrqz) [inline cloud/services/index.md:45]: Link to currently broken page:
https://docs.timescale.com/cloud/latest/services/#learn-more-about-timescaledb

**Comment by Rajakavitha1** [comment]
Hi @andy2mrqz !!! Thank you for raising the issue and also submitting a PR to resolve the issue. We truly appreciate your time and efforts in making the Timescale documentation better.

File: cloud/services/index.md [hunk 6]
```diff
@@ -42,7 +42,7 @@ For additional documentation on how to:
...
+[aggregates-info]: /getting-started/latest/create-cagg
```
Comment (Loquacity) [inline cloud/services/index.md:45]:
```suggestion
[aggregates-info]: /timescaledb/getting-started/:currentVersion:/create-cagg
```

File: getting-started/create-cagg/create-cagg-basics.md [hunk 7]
```diff
@@ -170,4 +170,4 @@ found in the [continuous aggregates docs][continuous-aggregates].
 [min]: https://www.postgresql.org/docs/current/tutorial-agg.html
 [first]: /api/:currentVersion:/hyperfunctions/first/
 [last]: /api/:currentVersion:/hyperfunctions/last/
-[cagg-policy]: /getting-started/create-cagg/create-cagg-policy/
+[cagg-policy]: /create-cagg/create-cagg-policy/
```
Comment (Loquacity) [inline getting-started/create-cagg/create-cagg-basics.md:173]:
```suggestion
[cagg-policy]: /timescaledb/getting-started/:currentversion:/create-cagg/create-cagg-policy/
```

File: timescaledb/quick-start/node.md [hunk 8]
```diff
@@ -519,7 +519,7 @@ Node application, be sure to check out these advanced TimescaleDB tutorials:
...
+[continuous-aggregates]: /getting-started/latest/create-cagg/
```
Comment (Loquacity) [inline timescaledb/quick-start/node.md:522]:
```suggestion
[continuous-aggregates]: /timescaledb/getting-started/:currentVersion:/create-cagg/
```

File: timescaledb/tutorials/analyze-cryptocurrency-data.md [hunk 9]
```diff
@@ -637,6 +637,6 @@ Ready for even more learning? Here's a few suggestions:
...
+[continuous-aggregates]: /getting-started/latest/create-cagg
```
Comment (Loquacity) [inline timescaledb/tutorials/analyze-cryptocurrency-data.md:640]:
```suggestion
[continuous-aggregates]: /timescaledb/getting-started/:currentVersion:/create-cagg
```

File: timescaledb/tutorials/nyc-taxi-cab.md [hunk 10]
```diff
@@ -815,7 +815,7 @@ Ready for more learning? Here's a few suggestions:
 [nyc_data]: https://timescaledata.blob.core.windows.net/datasets/nyc_data.tar.gz
 [postgis]: http://postgis.net/documentation
 [time-series-forecasting]: /tutorials/time-series-forecast/
-[continuous-aggregates]: /getting-started/create-cagg/
+[continuous-aggregates]: /getting-started/latest/create-cagg/
```
Comment (Loquacity) [inline timescaledb/tutorials/nyc-taxi-cab.md:818]:
```suggestion
[continuous-aggregates]: /timescaledb/getting-started/:currentversion:/create-cagg/
```

**Comment by Loquacity** [comment]
@charislam Can you please check my working here? 💖

**Comment by andy2mrqz** [comment]
@charislam @Loquacity thank you for reviewing!  Would you like me to commit the suggestions or will you handle that?  I'll let you take care of that if you were planning on it.

Also, is it recommended to use `:currentVersion:` everywhere in place of `latest`?  I noticed that in one example all the other links used `latest`, but the suggestion used `:currentVersion:`.

That seems to make sense, just wanted to make sure.  Maybe that's some more cleanup for down the line, changing other instances of `latest` in the docs to use `:currentVersion:` 🙂

**Comment by charislam** [comment]
Thank _you_ for the PR, @andy2mrqz! We'll take care of it, easy enough as we need to do the merge anyway! 🙂 

We do recommend `:currentVersion:`, and we probably need to fix that in some places! 😊 (There is one exception: within callouts you can't use `:currentVersion:` and have to use `latest`. When you get into the weeds, the site building can get ... complicated 😬)

**Review by Loquacity — APPROVED** [review]

**Pull request merged**